python/hail/matrixtable.py

```diff
@@ -582,19 +569,6 @@ def col_key(self):
             self._col_key = jiterable_to_list(self._jvds.colKey())
         return hail.struct(**{k: self[k] for k in self._col_key})
 
-    @property
-    def col_schema(self):
-        """The schema of column-indexed fields in the matrix.
-
-        Returns
-        -------
-        :class:`.tstruct`
-             Column schema.
-        """
-        if self._col_schema is None:
-            self._col_schema = Type._from_java(self._jvds.colType())
-        return self._col_schema
-
     @property
     def row_key(self):
         """The list of row key fields.
```

Comment:
the documentation for col_key, row_key, and probably partition key should not say they are the `list of`. I think I fixed it in my rename branch, but it might be better for you to fix it here.